UNRELEASED.md

```diff
@@ -2,6 +2,11 @@
 
 > Place your changes below this line.
 
+**Breaking:**
+ - Upgraded `bpk-tokens` to `35.0.1`. On Android, this changes `borderRadiusSm` from `4` to `8`. To keep using the old border radius (`4`), change your uses of `borderRadiusSm` to `borderRadiusXs`.
+
+**Fixed:**
+ - Cards and Panels now have a corner radius of `bpk-spacing-md` by default.
```

Comment:
```suggestion
 - The default corner radius of cards and panels has been increased from `bpkBorderRadiusXs` to `bpkBorderRadiusXs ` (4 to 12).
```

I think this is a bit more informative as it tells them it's increased, and from what to what.